### DISCUSSION TITLE
New implementation in DicomClient.

### DISCUSSION THREAD

**Discussion by DouglasMachas** [fo-dicom/fo-dicom#1814] [Q&A]
In a certain case, it was necessary to create a fo-dicom Client and define its local TCPClient IP.
We implemented a configuration in the DesktopNetworkStream class to force a local IP when creating the TCPClient:

![image](https://github.com/fo-dicom/fo-dicom/assets/147630415/df779817-283e-4a73-a36d-acc63b2b14a5)

Please, implement this functionality in new versions, so that it is possible to define the local IP of the TCPClient.
There are servers that do not accept more than one IP in the registration for reception, however there are cases where we need to upload more than one client to communicate with a server.

**Comment by jaime-olivares** [comment]
Question answered here: https://github.com/fo-dicom/fo-dicom/issues/1872
